### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by shyusu4**
**Created modal element with JavaScript**

- Used basic JavaScript syntax.
- Used JavaScript to manipulate DOM elements.
- Used JavaScript events.
- Used objects to store and access data.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -132,36 +132,51 @@ <h4>Skills</h4>
...
       <h3>MY RECENT WORK</h3>
       <hr />
-      <div class="popup-container">
-        <!-- <button id="open-popup"></button> -->
-      </div>
-      <div class="popup">
-        <div class="close-btn">&times;</div>
-        <span>Multi - Post stories</span>
-        <div class="popup-info">
+      <section class="modal" id="modal"></section>
```
Comment (muneebulrehman) [inline index.html:135]: Consider increasing the blur of the background so as to make the text easy to read.
This is your design now.
image

File: index.html [hunk 2]
```diff
@@ -132,36 +132,51 @@ <h4>Skills</h4>
...
+            <a href="#" class="modal-button"
+              >See Source
+              <img class="modal-icon" src="imgs/github.svg" alt="github icon" />
+            </a>
+          </div>
         </div>
       </div>
+      <div id="overlay"></div> -->
 
       <div class="flex-container">
```
Comment (muneebulrehman) [inline index.html:176]: Not only the modals but the whole work section should be generated dynamically using javascript. Consider removing the hardcoded HTML and generating the work section dynamically.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @shyusu4 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (shyusu4) [reply index.html:135]: Noted.

File: index.html (hunk 2 above)
Comment (shyusu4) [reply index.html:176]: Okay, will do that.

**Pull request merged**